### DISCUSSION TITLE
2023 (10) October US PST timezone community sync

### DISCUSSION THREAD

**Discussion by kerneltime** [apache/ozone#5453] [Community Sync Meetings]
# Oct 16th
## Recording
https://cloudera.zoom.us/rec/share/n-8ZDMprolvBoQxjlgheJTVtqTOVzWaxV3NJ-Pa99pgc8ObjGxSbSv_McLAnFTrg.myWXNmWZDzdjjDZW
Passcode: 8Jp.971n
# Topics

1. Pending PRs
2. New Jiras
3. CoC 2023 follow up
4. GRPC slowness discussion
5. Missing containers in Recon discussion

**Comment by jojochuang** [comment]
I don't know if any one had the same problem, but this morning I wasn't able to join the meeting. Not sure if it's because Cloudera enabled a few more protections for calls. Did any one else have the same trouble, and if we need to switch to a non-Cloudera zoom subscription.

**Reply by kerneltime** [reply]
Folks from GResearch were able to join. Maybe you need to be logged in? If others faced an issue I can work offline and try the same zoom link and figure out what is needed.
